backend/Services/CompanyDataBuilder.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Services;
+
+use Internships\Interfaces\SerializableInfo;
+use Internships\Models\Company;
+use Internships\Models\PathPair;
+
+class CompanyDataBuilder extends DataBuilder implements SerializableInfo
```

Comment:
It would be nice to test it out in unit tests.

Reply:
There's #9 task for that.

Reply:
Tests are in separate issues.  They will be implemented when I'll learn how to make them.